### ISSUE TITLE
Getting rid of "keyCode" dependency

### ISSUE THREAD

**Issue by photopea** [jaywcjlove/hotkeys#198]
The lates standards encourage users to not use the "keyCode" property of events.  https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Your library uses it. Do you think you could get rid of that dependency? Or do you think it is not possible to detect some key combinations without it?

**Comment by jaywcjlove** [comment]
Consider browser compatibility issues. @photopea

**Comment by photopea** [comment]
@jaywcjlove I think there are less than 0.1% of people with such old browsers (like 15 years or more). And these people probably do not expect to do anything "interactive" in their browsers anyway. It could make your library even smaller.

**Comment by jaywcjlove** [comment]
@photopea Yes, I want to make some changes in the next big version.

**Comment by jayankmayukh** [comment]
`KeyboardEvent.keyCode`, `KeyboardEvent.charCode` and `KeyboardEvent.which` are all deprecated now. I think you should keep those around for compatibility only and start using `KeyboardEvent.code` or `KeyboardEvent.key` if those are available in browser.

**Comment by BANOnotIT** [comment]
The main problem with `KeyboardEvent.key` is that it'll print different keys on different keyboard layouts (english `K` will be russian `Л` which makes no sense in i18n).

**Comment by nus3** [comment]
The npm package for testing, @testing-library/user-event, will be removed keyCode property on keyboard events in v14, so if you use this library, your tests will not succeed.

https://github.com/testing-library/user-event/releases/tag/v14.0.0

> Support for keyCode property on keyboard events has been removed.

@testing-library/user-event has 1.7k stars and is used by many people, so it may be a problem!

Is there anything I can do to help?

**Comment by Isti115** [comment]
I think that `KeyboardEvent.code` should be chosen, since that avoids having issues with hotkeys working differently while CapsLock is active.

**Comment by ogonkov** [comment]
Yeah, got this problem with jsdom + user-event@14, tests are failing because my cmd+enter hotkey is not triggered
